docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       retries: 5
 
   subquery-node:
-    image: onfinality/subql-node:latest
+    image: onfinality/subql-node:v1.13.3
```

Comment:
Do we need to go with this version in Dockerfile too?

Reply:
Yes. Will set it on confirm the POC for GCP otherwise delete the dockerfile